docs/intro/install.md

```diff
@@ -0,0 +1,6 @@
+# Installing Krustlet
+
+Right now, no official release is available for Krustlet. You will need to follow the
+[developer guide](../community/developers.md) to compile Krustlet from scratch.
```

Comment:
Consider adding a note about how there will be releases once we get to an initial stable point